config/300-vspherebinding.yaml

```diff
@@ -22,12 +21,21 @@ spec:
     shortNames:
     - vsb
   scope: Namespaced
-  subresources:
-    status: {}
-  additionalPrinterColumns:
-  - name: Ready
-    type: string
-    JSONPath: ".status.conditions[?(@.type=='Ready')].status"
-  - name: Reason
-    type: string
-    JSONPath: ".status.conditions[?(@.type=='Ready')].reason"
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        type: object
+        x-kubernetes-preserve-unknown-fields: true
+        # TODO: use controller-gen from controller-tools to fill this in?
```

Comment:
What's the current pattern in Knative projects to create schemas?

Reply:
we have been using this: https://github.com/knative/eventing/tree/main/cmd/schema

Reply:
tracking https://github.com/vmware-tanzu/sources-for-knative/issues/253